client/my-sites/customer-home/cards/features/grow-earn/index.jsx

```diff
@@ -35,9 +35,11 @@ export const GrowEarn = ( { siteSlug, expandToolsAndTrack, isSiteWPForTeamsProp
 
 	return (
 		<Card className="grow-earn">
-			<CardHeading>{ translate( 'Grow & Earn' ) }</CardHeading>
+			<CardHeading tagName="h2" size={ 16 }>
+				{ translate( 'Grow & earn' ) }
+			</CardHeading>
 			<h6 className="grow-earn__card-subheader customer-home__card-subheader">
-				{ translate( 'Grow your audience and earn money' ) }
+				{ translate( 'Grow your audience and earn money.' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.